src/main/java/org/breedinginsight/brapps/importer/model/imports/BrAPIImportService.java

```diff
@@ -33,6 +33,21 @@
 public abstract class BrAPIImportService {
     public String getImportTypeId() {return null;}
     public BrAPIImport getImportClass() {return null;}
+    public String getWrongDataTypeMsg(String columnName) {
```

Comment:
I think this method name should be changed to something like `getInvalidIntegerMsg`

Reply:
Done, 1fc3c81.